src/main/js/src/containers/ExamDates/ExamDateModal/ExamDateView.js

```diff
@@ -77,19 +77,19 @@ const ExamDateView = props => {
   const minPostAdmissionStartDate =
     registrationEndDate && moment(registrationEndDate).add(1, 'days').format('YYYY-MM-DD');
   const maxPostAdmissionStartDate =
-    postAdmissionEndDate && moment(postAdmissionEndDate).subtract(1, 'days').format('YYYY-MM-DD');
```

Comment:
Tänne tiedostoon tehty vain korjaukset virkailija-UI:hin että on ylipäänsä mahdollista luoda yhden vrk mittaisia jälki-ilmoittautumisaikoja. Backendillä ei ollut tämän osalta rajoitetta ja näin sinne ei tarvinnut koskea.